Stockchecker.js

```diff
@@ -84,4 +83,4 @@ function doesntexist(element, name) {
 
 
 
-console.log(doesntexist(product, 'spoon'));
\ No newline at end of file
+console.log(doesntexist(product, 'Table'));
```

Comment:
the function name doesntexist is not descriptive, an outsider can't understand what it means. i didn't before i read the comment above :) call it sth like deleteProductByName